src/formatMarkup.js

```diff
@@ -51,6 +51,23 @@ const ESCAPABLE_RAW_TEXT_ELEMENTS = Object.freeze([
   'title'
 ]);
 
+const getNonTextChildNode = (node) => {
+  for (let i = node.childNodes.length - 1; i >= 0; i--) {
+    if (!(defaultTreeAdapter.isTextNode(node.childNodes[i]) && (node.childNodes[i].value).trim() === '')) {
+      return node.childNodes[i];
```

Comment:
One idea per line.
```suggestion
    const childNode = node.childNodes[i];
    const isTextNode = defaultTreeAdapter.isTextNode(childNode);
    const isEmptyString = isTextNode && childNode.value.trim() === '';
    if (!isEmptyString) {
      return childNode;
```

Reply:
Since the code was not working properly, I did not prioritize formatting it correctly. However, I do realize I should have formatted it to make it more readable. Unfortunately, despite these changes, it isn't working as expected.

Reply:
I'm struggling to debug it. It's hard to see what changes actually happened that could be impacting the code. This would be simpler if the code refactoring was done in it's own PR, where all existing tests continue to pass.

I'll make a PR to move some of these changes out of this PR, I'll probably have time to do that tomorrow.